capstone/src/main/webapp/index.html

```diff
@@ -8,13 +8,14 @@
   </head>
   <body>
     <h2>Welcome to UniViz!</h2>
-    <form autocomplete="off">
+    <!-- TODO: This form needs the most work (need to get data async) -->
+    <form action="/search" autocomplete="off" type="GET" id="search-bar">
       <label for="search">Search for colleges:</label><br>
-      <input type="text" id="search" name="search" placeholder="College Name" oninput="giveSuggestions()" onkeydown="keyDown(event)"><br>
-      <input type="submit" value="See your colleges!">
+      <input type="text" id="search" name="query" placeholder="College Name" oninput="giveSuggestions()" onkeydown="keyDown(event)"><br>
```

Comment:
im not seeing this function in your js files? will it be implemented later or is there a naming inconsistency?

Reply:
I am trying to determine how to fit the async call to the API in here, and in doing say I've shuffled quite a bit around...I'd say it's probably a naming inconsistency

Reply:
also looks like giveSuggestions is also in my java file :)